src/model/data_extractor.py

```
@@ -13,62 +13,68 @@ class DataExtractor:
             df: pd.DataFrame = pd.read_excel(excel_path, header=0, engine="openpyxl")
-            
+
             # 列名の確認
-            expected_columns = ['time', 'value']
+            expected_columns = ["time", "value"]
             if list(df.columns[:2]) != expected_columns:
-                raise ValueError(f"1行目は 'time', 'value' という列名である必要があります。現在: {list(df.columns[:2])}")
-            
+                raise ValueError(
+                    f"1行目は 'time', 'value' という列名である必要があります。現在: {list(df.columns[:2])}"
+                )
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)